### PR TITLE
Add additional details to Ruby Workers architecture

### PR DESCRIPTION
**Pull request by zachmargolis**
Adds details based on questions from @mzia

### CODE REVIEW

File: _articles/appdev-ruby-worker-jobs.md [hunk 1]
```diff
@@ -64,6 +64,24 @@ The lifecycle of a job:
...
 
+### Client-Side Encryption
+
+The browser encrypts images before they are uploaded to S3. The browser generates 256-bit AES key,
+with a separate 12-byte random IV per image. It uses AES-GCM.
+
+### Server-Side Encryption
+
+The server stores job arguments in RDS. The Ruby code encrypts arguments that contain PII
+using the same encryption to encode a session: AES-256-GCM inside of and AWS KMS.
```
Comment (mitchellhenke) [inline _articles/appdev-ruby-worker-jobs.md:75]:
```suggestion
using the same encryption to encode a session: AES-256-GCM inside of AWS KMS.
```

**Review by mitchellhenke — APPROVED** [review]

File: _articles/appdev-ruby-worker-jobs.md [hunk 2]
```diff
@@ -64,6 +64,24 @@ The lifecycle of a job:
...
+The AES key is the application's `session_encryption_key`, stored with the application secrets.
```
Comment (mzia) [inline _articles/appdev-ruby-worker-jobs.md:76]: This is great. Can you define what "application secrets" are? or reference to something else that talks to application secrets? we might state  "application secrets are sensitive config items used by the login.gov application during run time and stored in S3/memory. etc."  
Trying to make this doc stand on its own.

Comment (zachmargolis) [reply _articles/appdev-ruby-worker-jobs.md:76]: got it, yup! will add that

Comment (zachmargolis) [reply _articles/appdev-ruby-worker-jobs.md:76]: Updated in 1491b67871aca0d31110760088e0dc08d86ad4c8

**Pull request merged**